src/Symfony/Component/DependencyInjection/Attribute/Memoizable.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Attribute;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class Memoizable
```

Comment:
IMO we can skip this class as attribute, useless double check in compiler pass :)

Reply:
I used it to make a first filter instead of retrieving all services' method attributes.

By the way, this attribute could contain default configuration like TTL or cache pool.

Reply:
Yes, good idea. ;)